src/components/organize/journeys/JourneyInstanceSidebar.tsx

```diff
@@ -0,0 +1,159 @@
+import { Add } from '@material-ui/icons';
+import { useState } from 'react';
+import { Button, ClickAwayListener, Divider, Grid } from '@material-ui/core';
+import { FormattedMessage as Msg, useIntl } from 'react-intl';
+
+import JourneyMilestoneProgress from 'components/organize/journeys/JourneyMilestoneProgress';
+import JourneyPerson from './JourneyPerson';
+import { MUIOnlyPersonSelect as PersonSelect } from 'components/forms/common/PersonSelect';
+import ZetkinSection from 'components/ZetkinSection';
+import {
+  ZetkinJourneyInstance,
+  ZetkinPerson as ZetkinPersonType,
+} from 'types/zetkin';
+
+const JourneyInstanceSidebar = ({
+  journeyInstance,
+  onAddAssignee,
+  onAddMember,
+  onRemoveAssignee,
+  onRemoveMember,
```

Comment:
I think these should be `onAddSubject` and `onRemoveSubject` (instead of "Member").